src/telliot_feeds/queries/query_catalog.py

```diff
@@ -118,7 +118,11 @@
 query_catalog.add_entry(
     tag="diva-protocol-example",
```

Comment:
the example query tag doesn't have a feed.

Reply:
Thanks for spotting this. Added one.